src/Command/Redis/ZDIFF.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Predis\Command\Redis;
+
+use Predis\Command\Command as RedisCommand;
+use Predis\Command\Traits\Numkeys;
+use Predis\Command\Traits\WithScores;
+use Predis\Command\Traits\Keys;
+
+/**
+ * @link https://redis.io/commands/zdiff/
+ *
+ * This command is similar to ZDIFFSTORE, but instead of
+ * storing the resulting sorted set, it is returned to the client.
+ */
+class ZDIFF extends RedisCommand
+{
+    use Numkeys {
+        Numkeys::setArguments as setNumkeys;
+    }
+    use WithScores {
+        WithScores::setArguments as setWithScore;
```

Comment:
Are these explicit overwrites needed?

Reply:
Yeah, both this traits contains overridden setArguments method so it can be used without any additional code changes in separate, but if we wan't to combine them we need to add explicit aliases so we have different aliases for different setArguments approach and call it within a main class setArgument. This way this traits becomes more flexible and reusable in my opinion

Reply:
And yeah, they're required, my IDE highlighted it